pricelist_per_product/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Akretion
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Pricelist Per Product",
+    "summary": "Display pricelist items in products",
+    "version": "9.0.1.0.0",
+    "category": "Sales",
+    "website": "https://laslabs.com",
+    "author": "Akretion, Laslabs, Odoo Community Association (OCA)",
```

Comment:
I think author is a bit of an overreach here too. From what I can tell, you didn't do much here. My general rule is a 30-40% logic addition/refactor deserves authorship. Version migrations, bugfixes, etc. for the most part you're just a contributor.